src/main/java/com/github/dockerjava/core/command/BuildImageCmdImpl.java

```diff
@@ -181,7 +190,10 @@ public Long getShmsize() {
     @Override
     public BuildImageCmdImpl withTag(String tag) {
         checkNotNull(tag, "Tag is null");
-        this.tag = tag;
+        if (this.tags == null) {
+            this.tags = new HashSet<>(4);
+        }
+        this.tags.add(tag);
```

Comment:
@KostyaSha is such version check already implemented somewhere in the code base or I have to write one?